src/components/icon/icon.jsx

```diff
@@ -21,7 +28,7 @@ export default function Icon(props) {
       role={props.role}
     >
       { props.title && <title>{props.title}</title> }
-      <use xlinkHref={`#${props.name}`} />
+      <use tabIndex={tabindex} onKeyDown={onKeyDown} xlinkHref={`#${props.name}`} />
```

Comment:
It's odd that the `onKeyDown` is not onthe same element as the `onClick`. Is there a particular reason for that? Normally, we want to keep them on the same element.